pages/iam/how-to/set-up-sso-with-authentik.mdx

```diff
@@ -0,0 +1,70 @@
+---
```

Comment:
Hello! Can you add this menu entry to `iam/menu.js` please? I tried pushing to a branch in your fork, but I don't have the rights.

Please add it under the "How to set up identity federation" entry
```
        {
          label: 'How to set up SSO with Authentik',
          slug: 'set-up-sso-with-authentik'
        },
 ```
 
 thank you!